mood-analysis.rb

```diff
@@ -19,6 +24,34 @@ def analyze_mood(words)
   return ":-|"
 end
 
+def happy_days(text)
+  smile_count = 0
+  day_count = 0
+  text.each do |line|
+    day_count += 1
+    if analyze_mood(line) == ":-)"
+      smile_count += 1
+    end
+    if smile_count == 3
+      break
```

Comment:
This if/break combo works, but there's no advantage to it over just putting the `return "It took...` here. That way you also wouldn't have to repeat the exact same if condition test.